src/main/java/simpli/core/Simpli.java

```diff
@@ -0,0 +1,110 @@
+package simpli.core;
+
+import simpli.configs.SimpliConfiguration;
+import simpli.exceptions.ActionException;
+import simpli.exceptions.TaskException;
+import simpli.interpreter.Interpreter;
+import simpli.parser.Parser;
+import simpli.storage.Storage;
+import simpli.tasks.TaskList;
+import simpli.ui.Ui;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Main chatbot.
+ */
+public class Simpli {
+    private Ui ui;
+    private TaskList taskList;
+    private Parser parser;
+    private Interpreter intrpr;
+    private Storage storage;
+
+    /**
+     * Initializes the chatbot and its components.
+     */
+    public Simpli() {
+        this.ui = new Ui();
+        this.taskList = new TaskList();
+        this.parser = new Parser();
+        this.intrpr = new Interpreter(ui, taskList);
+        this.storage = new Storage(parser, intrpr, taskList);
+    }
+
+    /**
+     * Starts running the chatbot.
+     */
+    public void start() {
+        // welcome message
+        greet();
+
+        try {
+            storage.loadTasksFromFile(SimpliConfiguration.DATAPATH);
+        } catch (IOException e) {
+            ui.display("The file is corrupted :(");
+        } catch (ActionException e) {
+            ui.display("There is something wrong with the data/simpli.csv file :(");
+        } catch (TaskException e) {
```

Comment:
I like how you extracted out two kinds of Simpli Exceptions to target unsupported action commands and invalid task parameters separately.